.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,30 @@
+name: docker-build
+on: [push]
+
+env:
+  PROJECT_ID: docker-images-180022
+  IMAGE_NAME: gcr.io/docker-images-180022/apps/samson_secret_puller
+
+jobs:
+  build_image:
+    strategy:
+      matrix:
+        ARCH: ["amd64", "arm64"]
+    runs-on:
+      [
+        "self-hosted",
+        "${{ matrix.ARCH == 'arm64' && 'zendesk-slim-arm' || 'zendesk-stable' }}",
+      ]
```

Comment:
self-hosted runners are not available to public repos. should use [github-hosted runners](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources) eg. `ubuntu-20.04`